code/modules/unit_tests/species_change_clothing.dm

```diff
@@ -0,0 +1,15 @@
+/datum/unit_test/species_change_clothing
+
+/datum/unit_test/species_change_clothing/Run()
+	// Test lizards as their own thing so we can get more coverage on their features
+	var/mob/living/carbon/human/human_to_lizard = allocate(/mob/living/carbon/human/dummy/consistent)
+	human_to_lizard.equipOutfit(/datum/outfit/job/assistant/consistent)
+	human_to_lizard.dna.features["legs"] = DIGITIGRADE_LEGS //you WILL have digitigrade legs
+
+	var/obj/item/human_shoes = human_to_lizard.get_item_by_slot(ITEM_SLOT_FEET)
+	human_shoes.supports_variations_flags = NONE //do not fit lizards at all costs.
```

Comment:
Can we instead just equip already existing shoes that will always not support digitigrade? Then don't equip assistant outfit since it's not necessary

Reply:
I'm aware downstreams like Skyrat makes all shoes digitigrade (from what goof said, anyway) so I thought this would be better to avoid that being a problem for them.

Reply:
Ok